manim/scene/scene.py

```diff
@@ -848,19 +854,30 @@ def handle_caching_wait(func):
         def wrapper(self, duration=DEFAULT_WAIT_TIME, stop_condition=None):
             self.revert_to_original_skipping_status()
             self.update_skipping_status()
+            if file_writer_config["skip_animations"]:
+                logger.debug(f"Skipping wait {self.num_plays}")
+                func(self, duration, stop_condition)
+                # If the animation is skipped, we mark its has as None. When sceneFileWriter will start combining partial movie files, it won't take into account None hashes.
```

Comment:
Line too long

Reply:
"we mark its has as None"
Its "has"? or its hash?

Reply:
Sorry for that. I meant hash.